pyodide-build/pyodide_build/out_of_tree/build.py

```diff
@@ -1,15 +1,61 @@
 import os
+import shutil
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from unearth.finder import PackageFinder, TargetPython
 
 from .. import common, pypabuild, pywasmcross
 
 
-def run(exports, args):
+def fetch_pypi_package(package_spec, destdir):
```

Comment:
`unearth` looks great. In `mkpkg.py`, there is also a function that downloads from PyPI, so it would be nice if we can merge it to one. (No action required for this PR).